### PR TITLE
[CIVIS-8421] Update Quality Gates setup page as we removed DD_BETA_COMMANDS_ENABLED

### PR DESCRIPTION
**Pull request by liashenko**
### What does this PR do? What is the motivation?
Update Quality Gates setup page as we removed DD_BETA_COMMANDS_ENABLED parameter for `datadog-ci gate`.

### Merge instructions

Merge is blocked by https://github.com/DataDog/datadog-ci/pull/1150

- [x] Please merge after reviewing

### CODE REVIEW

File: content/en/quality_gates/setup/_index.md [hunk 1]
```diff
@@ -81,7 +78,7 @@ based on the current pipeline context. By default, the command succeeds.
...
-- `--no-wait`: Skips the default time that the command waits for the events (tests, static analysis violations) to arrive to Datadog. The default wait time makes sure that the events are queryable in Datadog before the rules are executed, avoiding incorrect evaluations. If, in your pipeline, the job containing the `datadog-ci gate evaluate` command is called several minutes after the related events are sent to Datadog, you can opt to skip this waiting time by specifying the `--no-wait` flag. However, if used incorrectly, this flag may result in inaccurate rule evaluations.
+- `--no-wait`: Skips the default time that the command waits for the events (e.g. tests, static analysis violations) to arrive to Datadog. The default wait time makes sure that the events are queryable in Datadog before the rules are executed, avoiding incorrect evaluations. If, in your pipeline, the job containing the `datadog-ci gate evaluate` command is called several minutes after the related events are sent to Datadog, you can opt to skip this waiting time by specifying the `--no-wait` flag. However, if used incorrectly, this flag may result in inaccurate rule evaluations.
```
Comment (liashenko) [inline content/en/quality_gates/setup/_index.md:81]:
```suggestion
- `--no-wait`: Skips the default time that the command waits for the events (for example, tests, static analysis violations) to arrive to Datadog. The default wait time makes sure that the events are queryable in Datadog before the rules are executed, avoiding incorrect evaluations. If, in your pipeline, the job containing the `datadog-ci gate evaluate` command is called several minutes after the related events are sent to Datadog, you can opt to skip this waiting time by specifying the `--no-wait` flag. However, if used incorrectly, this flag may result in inaccurate rule evaluations.
```

**Review by alxch- — APPROVED** [review]

**Review by buraizu — APPROVED** [review]
Thanks, looks good overall and approving, but not merging since the PR is not marked as ready to merge. Please let me know if you need the PR merged as well.

File: content/en/quality_gates/setup/_index.md [hunk 2]
```diff
@@ -43,7 +43,7 @@ The following example shows how to create a static analysis rule that fails when
 
 ## Invoking quality gates
 
-Quality Gates, requires [`datadog-ci`][7] version `2.19.0` or later.
+Quality Gates, requires [`datadog-ci`][7] version `2.27.0` or later.
```
Comment (andrea-mosk) [inline content/en/quality_gates/setup/_index.md:46]:
```suggestion
Quality Gates requires [`datadog-ci`][7] version `2.27.0` or later.
```

**Comment by liashenko** [comment]
@buraizu the datadog-ci new release is out! We can merge this PR now, thanks!

**Pull request merged**